client/blocks/upwork-banner/index.jsx

```diff
@@ -102,7 +106,7 @@ class UpworkBanner extends PureComponent {
 					alt={ translate( 'Upwork' ) }
 					width={ 390 }
 					className="upwork-banner__image"
-					src={ safeImageUrl( '/calypso/images/themes-banner/illustration-builder-referral.svg' ) }
```

Comment:
Am I correct in assuming that the `safeImageUrl ` is unnecessary (particularly after this change)? I don't believe it's used anywhere else for this purpose, so I removed it.